website/docs/r/firewall.html.md

```diff
@@ -115,6 +115,14 @@ The following arguments are supported:
 
 * `labels` - (Optional) List of labels in the format [ "label1", "label2" ].
 
+## Timeouts
+
+Timeouts configuration options (in seconds):
+
+* `create` - (Default value is 0, it means the operation runs without a timeout) Used for Creating resource.
```

Comment:
@nvthongswansea forgot to update the docs for all resources.

Reply:
It would be great if we add an example of how to use timeouts.

Reply:
Done